### PR TITLE
PILOT-934

### PR DESCRIPTION
**Pull request by afmam**
## Summary

- Removed neo4j calls
- Updated lineage creation with file names in payload
- Updated unit tests

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-934

## Type of Change

Please delete options that are not relevant.

- [X] Refactor or reformatting

## Testing

Are there any new or updated tests to validate the changes?

- [X] Yes
- [ ] No

## Test Directions

(Additional instructions for how to run tests or validate functionality if not covered by unit tests)

### CODE REVIEW

File: app/models/models_lineage.py [hunk 1]
```diff
@@ -119,6 +121,8 @@ def creation_form_factory(post_form):
         my_form = CreationForm()
         my_form.input_geid = post_form.input_geid
         my_form.output_geid = post_form.output_geid
+        my_form.input_name = post_form.input_name
+        my_form.output_name = post_form.output_name
```
Comment (QXgu) [inline app/models/models_lineage.py:125]: L57, I think the `self._attribute_map` also need to be updated as well?

File: app/commons/atlas/lineage_manager.py [hunk 2]
```diff
@@ -48,8 +48,8 @@ async def create(self, creation_form: CreationForm, version='v1'):
         )
         self._logger.info(f'_________typenames is: {typenames}')
         # to atlas post form
-        input_file_name = await self.get_file_name_by_geid(creation_form.input_geid)
-        output_file_name = await self.get_file_name_by_geid(creation_form.output_geid)
+        input_file_name = creation_form.input_name
```
Comment (QXgu) [inline app/commons/atlas/lineage_manager.py:51]: Creation_form class seems does not have these two attributes

File: app/models/models_lineage.py (hunk 1 above)
Comment (afmam) [reply app/models/models_lineage.py:161]: Updated self._attribute_map

File: app/commons/atlas/lineage_manager.py (hunk 2 above)
Comment (afmam) [reply app/commons/atlas/lineage_manager.py:51]: Updated CreationForm class

**Comment by QXgu** [comment]
@afmam I think it would be able to update the geid to id cause we don't use the geid term any more.

**Comment by afmam** [comment]
> @afmam I think it would be able to update the geid to id cause we don't use the geid term any more.

Updated this

**Review by QXgu — APPROVED** [review]

**Pull request merged**